pom.xml

```diff
@@ -247,7 +247,7 @@ under the License.
                 </repository>
                 <repository>
                     <id>LocationTech - Third Party</id>
-                    <url>https://repo.locationtech.org/content/repositories/thirdparty/</url>
+                    <url>https://repo.eclipse.org/content/repositories/thirdparty/</url>
```

Comment:
This should be:  https://repo.eclipse.org/content/repositories/locationtech-thirdparty/

Reply:
Good catch!

Reply:
LGTM. Do you want to merge this?

Reply:
sure.  I'll try to do it today.